mlcolvar/cvs/cv.py

```diff
@@ -43,6 +43,8 @@ def __init__(
         # OPTIM
         self._optimizer_name = "Adam"
         self.optimizer_kwargs = {}
+        self._lr_scheduler_name = None
```

Comment:
Do we need to cache this? Since ``self.lr_scheduler_kwargs`` is a public attribute, I'd keep it in there the whole time in case someone decide to change ``'scheduler'``.

Reply:
sounds good